test/ProfilerTest.cpp

```diff
@@ -1,8 +1,16 @@
-#include "gtest/gtest.h"
-
-#include "pipert/Profiler.h"
+#include <arpa/inet.h>
 
 #include <cstdio>
+#include <thread>
+
+#include "gtest/gtest.h"
```

Comment:
Putting the test headers in front of anything else does make sense in this case.